First contribution! Created some shapes.

DavidJSolano opened the pull request:
Edited and created three new shapes under acmcsuf.com\static\assets\1st\

DavidJSolano committed:
Update settings.json

DavidJSolano committed:
Added some basic shapes.

First commit, created 3 shapes.

DavidJSolano committed:
Corrected a syntax error for 1st contribution.

DavidJSolano committed:
Repositioned the shapes

DavidJSolano committed:
Merge branch 'main' into add-davids700

DavidJSolano committed:
Merge branch 'main' into add-davids700

DavidJSolano committed:
Added .ts file so shapes appear in confetti

I forgot to export the shapes to the confetti lol.

EthanThatOneKid commented on .vscode/settings.json:
```diff
@@ -5,5 +5,76 @@
+    "typeindex": "cpp",
+    "typeinfo": "cpp",
+    "valarray": "cpp"
+  }
```
I am not sure how these changes got here lol. Go ahead and remove all your changes from this file since your PR does not require you to edit this file.

DavidJSolano replied:
I'm just as clueless as you are with how this got there, just fixed tho thanks for pointing it out.

EthanThatOneKid commented on src/lib/public/confetti/new-shapes.ts:
```diff
@@ -0,0 +1 @@
+export default 'url(/assets/1st/basic_shapes.svg)';
```
Nit: In acmcsuf.com, we tend to use [kabob case](https://en.wikipedia.org/wiki/Kebab_case) for storing our files.
```suggestion
export default 'url(/assets/1st/basic-shapes.svg)';
```

EthanThatOneKid reviewed (changes requested):
Thanks for the awesome shapes. I left you a couple of comments 😎

DavidJSolano committed:
Tweaks to first contribution.

Corrected the naming to include kebab case and removed the random code in settings.json.

EthanThatOneKid reviewed (approved):
Thanks for making the requested changes! LGTM 😎

EthanThatOneKid committed:
Merge branch 'main' into add-davids700

DavidJSolano committed:
Created new confetti piece.

Re-amped the confetti.
